src/testdrive/src/bin/testdrive.rs

```diff
@@ -374,6 +374,7 @@ async fn main() {
                 eprintln!("testdrive: maximum number of errors reached; giving up");
                 break;
             }
+            tokio::time::sleep(Duration::from_secs(1)).await;
```

Comment:
Can you drop a note in explaining this (idk what it is, or why it shouldn't be deleted if someone else PRs that).

Reply:
Nit: does the `// TODO` comment indicate that this still needs to be done (it seems to be right there), or that we should investigate some other way to do the same thing?